server/server.jsx

```diff
@@ -45,6 +46,37 @@ app.get("/get-initial-markers", async (req, res) => {
   }
 });
 
+app.post("/filters-params", async (req, res) => {
+  try {
+    let { position, places } = req.query;
+    position = JSON.parse(position);
+    var data = new FormData();
+    data.append('tags', places);
+    data.append('categories', '4');
+    data.append('c1', position.c1.toString());
+    data.append('c2', position.c2.toString());
+    data.append('c3', position.c3.toString());
+    data.append('c4', position.c4.toString());
+    const result = await myMizuClient.post(
+      "/api/taps/search?api_key=56b13329-6163-4884-afc8-b6839cd3f618&l=en&v=1",
```

Comment:
@alexminden could you change this to use config.apiKey?

Reply:
Thank you for noticing this, I have updated the url ([fbb2c1a](https://github.com/mymizu/mymizu-web/pull/21/commits/fbb2c1abf98a7a5c7593fde37578529d20b43745))